packages/lexical-playground/src/nodes/ExcalidrawNode/ExcalidrawComponent.tsx

```diff
@@ -41,32 +41,30 @@ export default function ExcalidrawComponent({
   const [isModalOpen, setModalOpen] = useState<boolean>(
     data === '[]' && editor.isEditable(),
   );
-  const imageContainerRef = useRef<HTMLImageElement | null>(null);
+  const imageContainerRef = useRef<HTMLDivElement | null>(null);
   const buttonRef = useRef<HTMLButtonElement | null>(null);
   const captionButtonRef = useRef<HTMLButtonElement | null>(null);
   const [isSelected, setSelected, clearSelection] =
     useLexicalNodeSelection(nodeKey);
   const [isResizing, setIsResizing] = useState<boolean>(false);
 
-  const $onDelete = useCallback(
+  const onDelete = useCallback(
```

Comment:
This is named `$onDelete` for a reason, since it calls dollar functions it must also be a dollar function. This rename should fail the lint
```suggestion
  const $onDelete = useCallback(
```